zombienet/tests/0007-full_node_warp_sync.toml

```diff
@@ -0,0 +1,62 @@
+[relaychain]
+default_image = "{{RELAY_IMAGE}}"
+default_command = "polkadot"
+default_db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/relaychain-0b03131ee21dd174ea695f9cbfaed84720b742d8.tgz?raw=true"
+
+chain = "rococo-local"
+
+  [[relaychain.nodes]]
+  name = "alice"
+  validator = true
+
+  [[relaychain.nodes]]
+  name = "bob"
+  validator = true
+
+[[parachains]]
+id = 2000
+cumulus_based = true
+chain_spec_path = "0007-warp-sync-parachain-spec.json"
+add_to_genesis = false
+
+  [[parachains.collators]]
+  name = "charlie"
+  validator = true
+  image = "{{COL_IMAGE}}"
+  command = "polkadot-parachain"
+  args = [""]
+  db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/parachain-c460b5f3e807c3a5c3093da4541d03d658cfa10c.tgz?raw=true"
+
+  [[parachains.collators]]
+  name = "dave"
+  validator = true
+  image = "{{COL_IMAGE}}"
+  command = "polkadot-parachain"
+  args = ["-lparachain=debug"]
+  db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/parachain-c460b5f3e807c3a5c3093da4541d03d658cfa10c.tgz?raw=true"
+
+  [[parachains.collators]]
+  name = "eve"
+  validator = true
+  image = "{{COL_IMAGE}}"
+  command = "polkadot-parachain"
+  args = ["-lparachain=debug"]
+  db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/parachain-c460b5f3e807c3a5c3093da4541d03d658cfa10c.tgz?raw=true"
+
+  [[parachains.collators]]
+  name = "ferdie"
+  validator = true
+  image = "{{COL_IMAGE}}"
+  command = "polkadot-parachain"
+  args = ["-lparachain=debug"]
+  db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/parachain-c460b5f3e807c3a5c3093da4541d03d658cfa10c.tgz?raw=true"
+
+  [[parachains.collators]]
+  name = "one"
+  validator = false
+  image = "{{COL_IMAGE}}"
+  command = "polkadot-parachain"
+  args = ["-lsync=debug","--sync warp"]
+  db_snapshot = "https://github.com/samelamin/cumulus/blob/chain-snapshots/docker/snapshots/parachain-to-sync-2daf880e656929e7c6680769990984998715c7ec.tgz?raw=true"
```

Comment:
@michalkucharczyk can you please upload these snapshots to our s3 or wherever we store them? Ty!

Reply:
done in: https://github.com/paritytech/cumulus/pull/1909/commits/d212069669891c7778034127050ca548160abcb9

urls:
https://storage.googleapis.com/zombienet-db-snaps/cumulus/0007-full_node_warp_sync/parachain-c460b5f3e807c3a5c3093da4541d03d658cfa10c.tgz
https://storage.googleapis.com/zombienet-db-snaps/cumulus/0007-full_node_warp_sync/parachain-to-sync-2daf880e656929e7c6680769990984998715c7ec.tgz
https://storage.googleapis.com/zombienet-db-snaps/cumulus/0007-full_node_warp_sync/relaychain-0b03131ee21dd174ea695f9cbfaed84720b742d8.tgz

Reply:
TY!

Reply:
Shall I update the PR with these locations?

Reply:
already done in https://github.com/paritytech/cumulus/pull/1909/commits/d212069669891c7778034127050ca548160abcb9